CalPatRec/src/AgnosticHelixFinder_module.cc

```diff
@@ -730,44 +730,37 @@ namespace mu2e {
 
     size_t sortStartIndex = 0;
 
-    cHit hit;
-    hit.circleError2 = 1.0;
-    hit.helixPhi = 0.0;
-    hit.helixPhiError2 = 0.0;
-    hit.helixPhiCorrection = 0;
-    hit.inHelix = false;
-    hit.used = false;
-    hit.isolated = false;
-    hit.averagedOut = false;
-    hit.notOnLine = true;
-    hit.uselessTripletSeed = false;
-    hit.notOnSegment = true;
-
     // push back stopping target if it is to be used
     if (_useStoppingTarget == true) {
+      cHit hit;
```

Comment:
For a future PR.  A better way to write these two lines to define a c'tor for cHit that takes just this argument.  

What's written is referred to as two phase construction.  The preferred idiom is single phase construction unless that's logically impossible.

Reply:
Ah, I see. I will change that in my next PR.